cmd/os-reliable.go

```diff
@@ -83,16 +83,17 @@ func renameAll(srcFilePath, dstFilePath string) (err error) {
 	}
 
 	if err = reliableRename(srcFilePath, dstFilePath); err != nil {
-		if isSysErrNotDir(err) {
+		switch {
+		case isSysErrNotDir(err):
 			return errFileAccessDenied
-		} else if isSysErrPathNotFound(err) {
+		case isSysErrPathNotFound(err):
 			// This is a special case should be handled only for
 			// windows, because windows API does not return "not a
 			// directory" error message. Handle this specifically here.
 			return errFileAccessDenied
-		} else if isSysErrCrossDevice(err) {
+		case isSysErrCrossDevice(err):
 			return fmt.Errorf("%s (%s)->(%s)", errCrossDeviceLink, srcFilePath, dstFilePath)
-		} else if os.IsNotExist(err) {
+		case os.IsNotExist(err):
 			return errFileNotFound
 		}
```

Comment:
Don't we need `default` here (although it was even also missing in the original code);
```
default:
    return nil, err
```